METADATOS/metaout/leer2.txt

```diff
@@ -0,0 +1 @@
+Esta es la carpeta en donde estan las salidas de los archivos JSON
```

Comment:
not sure if "Metaout" folder it's a good name, maybe change it for something like metadata_output or something like that